source/math/FloatEuclidTransform.ooc

```diff
@@ -43,6 +46,22 @@ FloatEuclidTransform: cover {
 		this rotation == other rotation &&
 		this scaling == other scaling
 	}
+	convolveCenter: static func (euclidTransforms: VectorList<This>, kernel: FloatVectorList) -> This {
+		result := This new()
+		if (euclidTransforms count > 0) {
+			result scaling = 0.0f
+			quaternions := VectorList<Quaternion> new(euclidTransforms count)
+			for (i in 0 .. euclidTransforms count) {
+				result translation = result translation + euclidTransforms[i] translation * kernel[i]
+				result scaling = result scaling + euclidTransforms[i] scaling * kernel[i]
+				rotation := euclidTransforms[i] rotation
+				quaternions add(rotation _quaternion)
```

Comment:
Nope. Tried a bunch of stuff to get that to work but the compiler doesn't accept it. 
Tried quaternions add((euclidTransforms[i] rotation) _quaternion), quaternions add((euclidTransforms[i] rotation) as FloatRotation3D _quaternion), same error.